bin/Translation/translation_dumper.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Extract translation information from a JS file.
+ *
+ * Usage: pass the path of a JS file as a parameter.
+ *
+ * The script will output JSON containing the translation information on the standard output.
+ * Example of output:
+ * `{"translationsFound":[{"key":"mykey","domain":"mydomain"}]}`
+ *
+ */
+
+var walk = require( 'estree-walker' ).walk,
+    acorn = require( 'acorn' ),
```

Comment:
question: did you switch to acorn because estree-walker was not working with espree or just to follow the example in the README ?